x-pack/plugins/alerting/public/pages/maintenance_windows/components/create_maintenance_windows_form.tsx

```diff
@@ -39,28 +41,39 @@ export const useTimeZone = (): string => {
 };
 
 export const CreateMaintenanceWindowForm = React.memo<CreateMaintenanceWindowFormProps>(
```

Comment:
I wonder if we should rename this component to `MaintenanceWindowForm` now that it's being used for both creation and editing.

Also looking at this component again....curious to know what you think, maybe the component can just be a dumb component without knowing whether or not it's creating or editing, it will just work with whatever initial data it is given. If the user submits, in line 62-65 we can just call an `onSubmit` callback and delegate the HTTP call to the parent. So we don't have to thread through `onSuccess` and stuff.

Reply:
Yes I agree, I think this would be a good follow on task. I can make an issue!

Reply:
Created an issue here https://github.com/elastic/kibana/issues/155538